### PR TITLE
PAAS-1708: Add 'haproxy_cfg_timeout_server' config envvar

### PR DESCRIPTION
**Pull request by kevinvandel**
JIRA issue: https://jira.jahia.org/browse/PAAS-1708

### CODE REVIEW

File: mixins/haproxy.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ actions:
     - setSudoer: bl
     - cmd[bl]: yum install -y jq
       user: root
+    - env.control.AddContainerEnvVars[bl]:
+      vars:
+        auth_basic_enabled: "false"
+        haproxy_cfg_timeout_server: "5m"
```
Comment (laurentfufu) [inline mixins/haproxy.yml:16]: why not put this directly on node creation step in bl nodegroup definition  ?

File: packages/jahia/migrations/migrate-to-v8.yaml [hunk 2]
```diff
@@ -106,11 +106,15 @@ actions:
...
+    - env.control.AddContainerEnvVars[bl]:
+      vars:
+        haproxy_cfg_timeout_server: "5m"
     - cmd[bl]: |-
         HAPROXY_SYSCONFIG=/etc/sysconfig/haproxy
         HAPROXY_RELOAD=/usr/local/bin/haproxy-reload.sh
         curl -fLSso $HAPROXY_SYSCONFIG ${globals.repoRootUrl}/assets/haproxy/haproxy-sysconfig || exit 1
         curl -fLSso $HAPROXY_RELOAD ${globals.repoRootUrl}/assets/haproxy/haproxy-reload.sh || exit 1
         chmod +x $HAPROXY_RELOAD
+        service haproxy reload
```
Comment (laurentfufu) [inline packages/jahia/migrations/migrate-to-v8.yaml:119]: update assets/haproxy/haproxy-jahia-cloud.cfg is missing here

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/haproxy.yml (hunk 1 above)
Comment (kevinvandel) [reply mixins/haproxy.yml:16]: you should ask that to the one who added the definition of `auth_basic_enabled` here :stuck_out_tongue: 
I'll move both of them

Comment (kevinvandel) [reply mixins/haproxy.yml]: done

File: packages/jahia/migrations/migrate-to-v8.yaml (hunk 2 above)
Comment (kevinvandel) [reply packages/jahia/migrations/migrate-to-v8.yaml]: done

**Review by laurentfufu — APPROVED** [review]

**Comment by laurentfufu** [comment]
only need rebase :stuck_out_tongue:

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**